flytekit/models/core/identifier.py

```diff
@@ -62,6 +62,13 @@ def version(self):
         """
         return self._version
 
+    @property
+    def is_empty(self):
```

Comment:
oh i meant you could put this in FlyteIdlEntity and then do something like:

```
@property 
def is_empty(self):
    return len(self.to_flyte_idl().SerializeToString()) == 0
```

Reply:
gotcha. thanks.